root/programs/MultiPurposeAuthSite/MultiPurposeAuthSite/Controllers/ErrorController.cs

```diff
@@ -0,0 +1,214 @@
+﻿//**********************************************************************************
+//* Copyright (C) 2007,2016 Hitachi Solutions,Ltd.
+//**********************************************************************************
+
+#region Apache License
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. 
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+#endregion
+
+//**********************************************************************************
+//* クラス名        ：ErrorController
+//* クラス日本語名  ：エラー処理用コントローラ
+//*
+//* 作成日時        ：－
+//* 作成者          ：生技
+//* 更新履歴        ：
+//*
+//*  日時        更新者            内容
+//*  ----------  ----------------  -------------------------------------------------
+//*  2015/08/27  Supragyan         Created ErrorController class to display error messages and informations
+//*  2015/09/03  Supragyan         Rename Position data class to Exception data class
+//*  2015/09/03  Supragyan         Modified Index Action method
+//*  2015/09/04  Supragyan         Modified ArrayList to List of ExceptionData on Index action method
+//**********************************************************************************
+
+using System;
+using System.Web.Mvc;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+
+using Touryo.Infrastructure.Framework.Util;
+using Touryo.Infrastructure.Public.Str;
+
+namespace MultiPurposeAuthSite.Controllers
+{
+    #region ErrorController
+    /// <summary>
+    /// ErrorController class
+    /// </summary>
+    public class ErrorController : Controller
+    {
+        /// <summary>Form情報</summary>
+        private List<PositionData> list_form = new List<PositionData>();
+
+        /// <summary>Session情報</summary>
+        private List<PositionData> list_session = new List<PositionData>();
+
+        #region Index
+        /// <summary>
+        /// Index Action method to display an error message error information on the screen
+        /// </summary>
+        /// <returns>ActionResult</returns>
+        public ActionResult Index()
+        {
+            //画面にエラーメッセージ・エラー情報を表示する-----------------------------
+
+            // To get an error message from Session
+            string err_msg = (string)Session[FxHttpContextIndex.SYSTEM_EXCEPTION_MESSAGE];
```

Comment:
@daisukenishino 

Add the processing to check that session variable is not null.